src/clj/game/cards/basic.clj

```diff
@@ -82,14 +84,13 @@
                 :effect (req (play-instant state :corp (assoc eid :source :action :source-type :play)
                                            target {:base-cost [:click 1]}))}
                {:label "Advance 1 installed card"
-                :cost [:click 1 :credit 1]
                 :async true
                 :msg (msg "advance " (card-str state target))
-                :req (req (can-advance? state side target))
-                :effect (effect (update-advancement-requirement target)
-                                (add-prop (get-card state target) :advance-counter 1)
-                                (play-sfx "click-advance")
-                                (effect-completed eid))}
+                :req (req (and (can-advance? state side target)
+                               (can-pay? state side
+                                         (assoc eid :source :action :source-type :advance)
+                                         card nil [:credit 1 :click 1])))
```

Comment:
Does this cover when there are additional costs on the click ability?

Reply:
Can you make an example of a card creating additional costs on the click ability? I couldn't find any.. :/

Reply:
[Enhanced Login Protocol](https://netrunnerdb.com/en/card/06022) changes the cost of a basic action:

> As an additional cost to take the basic action to run a server for the first time each turn, the Runner must spend click.

It's true that a card doesn't exist that changes the "advance a card" basic action. But we should still be writing abilities like those cards do exist.